### PR TITLE
Add mcrute to cloudprovider related owners files

### PR DESCRIPTION
**Pull request by mcrute**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Adding myself (@mcrute) as an owner to cloud-provider as a sig-cloud-provider lead and maintainer of this code.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```
/sig cloud-provider
/cc @andrewsykim @cheftako

### CODE REVIEW

File: pkg/cloudprovider/providers/aws/OWNERS [hunk 1]
```diff
@@ -6,6 +6,7 @@ approvers:
 - gnufied
 - jsafrane
 - micahhausler
+- mcrutet
```
Comment (andrewsykim) [inline pkg/cloudprovider/providers/aws/OWNERS]: typot

**Review by cheftako — APPROVED** [review]
/lgtm

File: pkg/cloudprovider/providers/aws/OWNERS [hunk 2]
```diff
@@ -6,6 +6,7 @@ approvers:
 - gnufied
 - jsafrane
 - micahhausler
+- mcrute
```
Comment (andrewsykim) [inline pkg/cloudprovider/providers/aws/OWNERS]: +1 from me, @justinsb any objections?

**Pull request merged**